code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -247,6 +247,7 @@
 	colored_disk_mouth = mutable_appearance(SSgreyscale.GetColoredIconByType(/datum/greyscale_config/carp/disk_mouth, greyscale_colors), "disk_mouth")
 	ADD_TRAIT(src, TRAIT_DISK_VERIFIER, INNATE_TRAIT) //carp can verify disky
 	ADD_TRAIT(src, TRAIT_ADVANCEDTOOLUSER, INNATE_TRAIT) //carp SMART
```

Comment:
the intention of giving cayenne advancedtooluser was for the stripping, so it should probably be removed if theres just a trait for stripping now

Reply:
carp SMART

Reply:
you cant use comments i wrote against me

Reply:
carp SMART